io/index.js

```diff
@@ -15,20 +16,98 @@ async function retrieveGraphData(){
   const {numberOfDataPoints, dataPoints} = await loadJson('./data.json');
   console.log('numberOfDataPoints', numberOfDataPoints);
   console.log('dataPoints', dataPoints);
-  
+
   return dataPoints;
 }
 
 
+/**
+ * Function that allows usr to add new data points to current
+ * json data
+ */
+function getNewDataPoint(){
```

Comment:
This function is the right approach to the problem. 
So the next step would be to load the return the result JSON in the `retrieveGraphData` function above this.

The other guys are going to call this function and get: the data points, and any settings (that will later be added to your `data.json`). 
So that function will be the main entry point to using your work. The usage of this file will basically be:
``` js
const getData = require('../io/');

getData().then((settingsAndData)=>{
  const {dataPoints, ...settings} = settingsAndData;
  // do something with dataPoints and settings
})
```